controllers/userController.js

```diff
@@ -80,23 +72,22 @@ module.exports = {
       res.status(500).json({ message: "An login error occurred" });
     }
   },
-
   getUser: async (req, res) => {
     try {
-      const userId = req.params.id;
-      console.log(userId);
-      const user = await User.findOne({
-        where: {
-          id: userId,
-        },
+      const { id } = req.headers;
```

Comment:
why does getUser have an id in headers? According to the assignment user data will be come from token, do you know if you are setting id in from that? if yes, set it into body, not into headers. @akaashinno